web-console/next.config.js

```diff
@@ -1,11 +1,12 @@
 
 const UnpluginIcons = require("unplugin-icons/webpack")
 const { FileSystemIconLoader } = require("unplugin-icons/loaders")
+const CopyPlugin = require("copy-webpack-plugin")
 
 /** @type {import("next").NextConfig} */
 module.exports = {
   output: 'export', // https://nextjs.org/docs/app/building-your-application/deploying/static-exports
-  distDir: process.env.BUILD_DIR || 'out',
```

Comment:
why is this .next now?

Reply:
I am not sure why it was 'out' in the first place, but after an update to latest Next.js 14.1.0 the development build breaks when mixing 'out' and '.next'. 'out' is still used for a production build